.github/workflows/on-merge.yml

```
@@ -4,6 +4,9 @@ on:
       - closed
       - labeled
 
+env:
+  NODE_ENV: kibana-github-action
```

Would it be better with a backport specific env variable?
```suggestion
  BACKPORT_ENVIRONMENT: kibana-github-action
```

---

:+1:  while probably not relevant here, I think it makes sense to leave `NODE_ENV=(production|development)`